portal/eproms/views.py

```diff
@@ -255,3 +257,88 @@ def website_consent_script(patient_id):
         terms=terms, top_organization=top_org,
         entry_method=entry_method, redirect_url=redirect_url,
         declaration_form=declaration_form, patient_id=patient_id)
+
+
+def get_asset(uuid):
+    url = "{LR_ORIGIN}/c/portal/truenth/asset/detailed?uuid={uuid}".format(
+        LR_ORIGIN=current_app.config["LR_ORIGIN"], uuid=uuid)
+    data = requests.get(url).content
+    return json.JSONDecoder().decode(data)['asset']
+
+
+def get_any_tag_data(anyTags):
+    """
+        query LR based on any tags
+        this is an OR condition
+        will match any tag specified
+        e.g. anyTag=tag1&anyTag=tag2
+    """
+    tags_list = []
```

Comment:
This could a little cleaner using [some functions](https://docs.python.org/3.4/library/urllib.parse.html#urllib.parse.urlencode) from the `requests` module:
```
liferay_qs_params = {
    'anyTags': anyTags,
    'sort': 'true',
    'sortType': 'DESC',
}
liferay_url = "".join(
    current_app.config["LR_ORIGIN"],
    "/c/portal/truenth/asset/query?",
    requests.compat.urlencode(
        lr_qs_params,
        doseq=True,
    ),
)
```